--- bucardo.schema
+++ bucardo.schema
@@ -135,9 +135,9 @@ log_showpid|1|Show PID in the log output?
 log_showlevel|0|Show log level in the log output?
 log_showline|0|Show line number in the log output?
 log_showtime|3|Show timestamp in the log output?  0=off  1=seconds since epoch  2=scalar gmtime  3=scalar localtime
-log_conflict_file|bucardo_conflict.log|Name of the conflict detail log file
+log_conflict_file|/var/log/bucardo/bucardo_conflict.log|Name of the conflict detail log file
 log_level|NORMAL|How verbose to make the logging. Higher is more verbose.
-warning_file|bucardo.warning.log|File containing all log lines starting with "Warning"
+warning_file|/var/log/bucardo/bucardo.warning.log|File containing all log lines starting with "Warning"
 \.
 
 -- Versioning
@@ -161,7 +161,7 @@ email_debug_file||File to save a copy of all outgoing emails to
 flatfile_dir|.|Directory to store the flatfile output inside of
 host_safety_check||Regex to make sure we don't accidentally run where we should not
 piddir|/var/run/bucardo|Directory holding Bucardo PID files
-reason_file|bucardo.restart.reason.txt|File to hold reasons for stopping and starting
+reason_file|/var/log/bucardo/bucardo.restart.reason.txt|File to hold reasons for stopping and starting
 semaphore_table|bucardo_status|Table to let apps know a sync is ongoing
 statement_chunk_size|10000|How many primary keys to shove into a single statement
 stats_script_url|http://www.bucardo.org/|Location of the stats script
