--- bucardo.schema
+++ bucardo.schema
@@ -140,10 +140,10 @@
 log_showlevel|0|Show log level in the log output?
 log_showline|0|Show line number in the log output?
 log_showtime|3|Show timestamp in the log output?  0=off  1=seconds since epoch  2=scalar gmtime  3=scalar localtime
-log_conflict_file|bucardo_conflict.log|Name of the conflict detail log file
+log_conflict_file|/var/log/bucardo/bucardo_conflict.log|Name of the conflict detail log file
 log_showsyncname|1|Show the name of the sync next to the 'KID' prefix
 log_level|NORMAL|How verbose to make the logging. Higher is more verbose.
-warning_file|bucardo.warning.log|File containing all log lines starting with "Warning"
+warning_file|/var/log/bucardo/bucardo.warning.log|File containing all log lines starting with "Warning"
 \.
 
 -- Versioning
@@ -169,7 +169,7 @@
 isolation_level|repeatable read|Default isolation level: can be serializable or repeatable read
 piddir|/var/run/bucardo|Directory holding Bucardo PID files
 quick_delta_check|1|Whether to do a quick scan of delta activity
-reason_file|bucardo.restart.reason.txt|File to hold reasons for stopping and starting
+reason_file|/var/log/bucardo/bucardo.restart.reason.txt|File to hold reasons for stopping and starting
 semaphore_table|bucardo_status|Table to let apps know a sync is ongoing
 statement_chunk_size|8000|How many primary keys to shove into a single statement
 stats_script_url|http://www.bucardo.org/|Location of the stats script
